src/status_im2/contexts/quo_preview/switcher/group_messaging_card.cljs

```diff
@@ -0,0 +1,127 @@
+(ns status-im2.contexts.quo-preview.switcher.group-messaging-card
+  (:require [quo.react-native :as rn]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [status-im2.common.resources :as resources]))
+
+(def descriptor
+  [{:label "Title"
+    :key   :title
+    :type  :text}
+   {:label   "Status"
+    :key     :status
+    :type    :select
+    :options [{:key   :read
+               :value :read}
+              {:key   :unread
+               :value :unread}
+              {:key   :mention
+               :value :mention}]}
+   {:label "Counter Label"
+    :key   :counter-label
+    :type  :text}
+   {:label   "Type"
+    :key     :type
+    :type    :select
+    :options [{:key   :message
+               :value :text}
+              {:key   :photo
+               :value :photo}
+              {:key   :sticker
+               :value :sticker}
+              {:key   :gif
+               :value :gif}
+              {:key   :audio
+               :value :audio}
+              {:key   :community
+               :value :community}
+              {:key   :link
+               :value :link}
+              {:key   :code
+               :value :code-snippet}]}
+   {:label "Last Message"
+    :key   :last-message
+    :type  :text}
+   (preview/customization-color-option)])
+
+;; Mock data
+(def sticker {:source (resources/get-mock-image :sticker)})
+(def community-avatar {:source (resources/get-mock-image :community-logo)})
+(def gif {:source (resources/get-mock-image :gif)})
+(def coinbase-community (resources/get-mock-image :coinbase))
+(def photos-list
+  [{:source (resources/get-mock-image :photo1)}
+   {:source (resources/get-mock-image :photo2)}
+   {:source (resources/get-mock-image :photo3)}
+   {:source (resources/get-mock-image :photo1)}
+   {:source (resources/get-mock-image :photo2)}
+   {:source (resources/get-mock-image :photo3)}])
+
+(defn get-mock-content
+  [data]
+  (case (:type data)
+    :message
+    {:text (:last-message data)}
+
+    :photo
+    {:photos photos-list}
+
+    :sticker
+    sticker
+
+    :gif
+    gif
+
+    :audio
+    {:duration "00:32"}
+
+    :community
+    {:community-avatar coinbase-community
+     :community-name   "Coinbase"}
+
+    :link
+    {:source (resources/get-mock-image :status-logo)
+     :text   "Rolling St..."}
+
+    :code
```

Comment:
We might as well remove the check for `:code` because this is covered by the default return value.